app/assets/javascripts/application.js

```diff
@@ -31,5 +31,6 @@
 //= require jquery.webui-popover/jquery.webui-popover.min
 //= require redactor/redactor.min
 //= require jquery-textcounter/textcounter.min
+//= require tophat_space.coffee
```

Comment:
Does this need to be added? I think it should be automatically picked up with the `require_tree` below.